include/core/lethe_grid_tools.h

```diff
@@ -37,7 +37,7 @@ namespace LetheGridTools
    *
    * @param dof_handler DofHandler of the triangulation on which to create the map
    *
-   * @param vertices_cell_map Object to which the data will be written
+   * @param vertices_cell_map The map of vertex ids to the set of neighbor cells
```

Comment:
I would name it "map container with vertex ids per cells in a set of neighbors"

Reply:
For me that's not the same since it's the global vertex ids to the set of cells that contains this vertex.